### DISCUSSION TITLE
已经续费了两年专业版，但是界面显示的还是快到期了

### DISCUSSION THREAD

**Discussion by favoritejelly** [getlantern/lantern-forum-cn#893] [Q&A]
您好！已经续费了两年专业版，但是界面显示的还是快到期了，请帮忙联系解决，谢谢！
![Screenshot_2024-09-25-21-20-51-953_com eg android AlipayGphone](https://github.com/user-attachments/assets/66373181-94cc-4fa7-a189-6e00dba17afe)

**Comment by favoritejelly** [comment]
邮箱账号地址是sunxiaofeiabc@163.com

**Comment by MaxMKMax** [comment]
@favoritejelly OK 没问题. 我已经帮您提交上去了. 请您放心! 谢谢您支持蓝灯!

**Reply by favoritejelly** [reply]
Mike您好！已经续费成功了，但是提前续费的并没有提供多3个月的使用时间，烦请帮忙再提交下，谢谢！
